code/WorkInProgress/JobXPRewards.dm

```diff
@@ -114,13 +114,21 @@ mob/verb/checkrewards()
 	desc = "A bucket! And it's red! Wow."
 	required_levels = list("Janitor"=5)
 	claimable = 1
-	claimPerRound = 1
```

Comment:
why remove the per round - for both

Reply:
> why remove the per round - for both

I assume because "making a bucket/mop out of thin air should be limited per round, whereas making your bucket/mop look different isn't really abusable".